Modules/Package.swift

```diff
@@ -50,6 +50,7 @@ let package = Package(
         .package(url: "https://github.com/Automattic/wordpress-rs", revision: "alpha-20241116"),
         .package(url: "https://github.com/wordpress-mobile/GutenbergKit", revision: "4476d597b271778d001a26c50d74e527b54ebfef"),
         .package(url: "https://github.com/Automattic/color-studio", branch: "trunk"),
+        .package(url: "https://github.com/allenhumphreys/AztecEditor-iOS", branch: "ah/spm"),
```

Comment:
👋 @mokagio - do we want to point Aztec to a fork? If I understand correctly this will block us if we need to push out the rare Aztec fix. Or maybe there's a timeline to get back to the WP org repo shortly after this change?

Reply:
Agreed. This was in the PR that I based my work on, and then I forgot to follow up.

I'll get started porting the changes from that fork into the official repo.

Reply:
Sounds good!

Reply:
Closing the loop: https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1406